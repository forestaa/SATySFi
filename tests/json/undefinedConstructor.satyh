let x = P 32
