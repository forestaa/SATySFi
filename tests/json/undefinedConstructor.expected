{
  "file": "undefinedconstructor.satyh",
  "range": {
    "start": { "line": 1, "character": 8 },
    "end": { "line": 1, "character": 12 }
  },
  "message": "! [Type Error] undefined constructor 'P'."
}
